### PR TITLE
Fix AudioContext doc typo

### PR DESCRIPTION
**Pull request by rvadhavk**

### CODE REVIEW

File: docs/api/en/audio/AudioContext.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h3>[method:AudioContext getContext]()</h3>
 		otherwise set it to a new [link:https://developer.mozilla.org/en-US/docs/Web/API/AudioContext AudioContext].
 		</p>
 
-		<h3>[method:AudioContext setContext]( [param:AudioConetxt value] )</h3>
+		<h3>[method:AudioContext setContext]( [param:AudioContext value] )</h3>
```
Comment (Mugen87) [inline docs/api/en/audio/AudioContext.html:32]: Can you please fix this issue for the Chinese version, too? 

https://github.com/mrdoob/three.js/blob/36524a4c6667e88e128f78d78b7a29acf80e46d8/docs/api/en/audio/AudioContext.html#L32

Comment (rvadhavk) [reply docs/api/en/audio/AudioContext.html:32]: Done.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Comment by zlf2002171** [comment]
unsubscribe

2020-05-18 

zlf2002171 



发件人：rvadhavk <notifications@github.com>
发送时间：2020-05-01 01:31
主题：Re: [mrdoob/three.js] Fix AudioContext doc typo (#19264)
收件人："mrdoob/three.js"<three.js@noreply.github.com>
抄送："Subscribed"<subscribed@noreply.github.com>

@rvadhavk commented on this pull request.



In docs/api/en/audio/AudioContext.html:
> @@ -29,7 +29,7 @@ <h3>[method:AudioContext getContext]()</h3>
 		otherwise set it to a new [link:https://developer.mozilla.org/en-US/docs/Web/API/AudioContext AudioContext].
 		</p>
 
-		<h3>[method:AudioContext setContext]( [param:AudioConetxt value] )</h3>
+		<h3>[method:AudioContext setContext]( [param:AudioContext value] )</h3>

Done.

**Pull request merged**